vite.config.ts

```diff
@@ -65,7 +65,7 @@ export default defineConfig(({ command }) => ({
       exclude: ['./.cache/**/*']
     }),
     wyw({
-      exclude: ['./.cache/**/*'],
+      exclude: ['./.cache/**/*', '**/*.d.ts', '**/*.gen.ts'],
```

Comment:
Do we not need to excluded `'**/*.d.ts', '**/*.gen.ts'` in `rolldown.config`?

Reply:
I did ad `'**/*.d.ts'`, as otherwise the build fails with the new plugin.
`'**/*.gen.ts'` is probably not needed as rolldown won't touch the website files.